AlphaWallet/InCoordinator.swift

```diff
@@ -225,7 +225,10 @@ class InCoordinator: NSObject, Coordinator {
 
     private func oneTimeCreationOfOneDatabaseToHoldAllChains() {
         let migration = MigrationInitializer(account: wallet)
-        migration.oneTimeCreationOfOneDatabaseToHoldAllChains(assetDefinitionStore: assetDefinitionStore)
+
+        DispatchQueue.global().async {
```

Comment:
We can't move the call for `oneTimeCreationOfOneDatabaseToHoldAllChains` to the background because it'll be very fragile. All other database access would fail or return wrong results until this is complete, and the database might be in an invalid state. I think the function is mostly a no-op if there's nothing to migrate and the migration only runs at most once, and only for users who started using the app a very long time ago. So maybe no need to optimize this.

Reply:
yep, ok i got it.